generate_html.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+
+import duckdb
+import pandas as pd
+from jinja2 import Environment, FileSystemLoader
+
+TEMPLATES_PATH = "templates"
```

Comment:
Could you make this a path relative to this file so that it will work when we execute this script from different locations?